src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonHighlightClickedBlocks.kt

```diff
@@ -36,6 +37,12 @@ object DungeonHighlightClickedBlocks {
         "§cThat chest is locked!",
     )
 
+    private const val WATER_ROOM_ID = "-60,-60"
+
+    private val inWaterRoom by RecalculatingValue<Boolean>(1.seconds) {
```

Comment:
since this is using the scoreboard, please work with a scroeboard update event instead to avoid overly long delays (scoreboard updates not instnatly while changing rooms, and having one additonal second delay could cause wrong values for longer time)

Reply:
should i add this to DungeonAPI? and just have this inWaterRoom val check if the room id stored in dungeon api is the water room id

Reply:
Yes, please!